File: day1/Quiz3/solution.java
public class solution {

    public static void main(String[] args) {
        pinChk("0000");
    } 

    public static void pinChk(String pinNum) {

        int strChk = pinNum.replaceAll("[0-9]", "").length();
        int numChk = pinNum.replaceAll("[^0-9]", "").length();
        System.out.println("strChk : " + strChk);
        System.out.println("numChk : " + numChk);
        if( strChk == 0 && ( numChk == 4 || numChk == 6 ) ){
            System.out.println(true);
        } else {
            System.out.println(false);
        }
        
    }

}

File: day1/Quiz3/solution.py
#"1234"  // true
#"12345" // false
#"a234"  // false
#"0000"    // true
#"1111"    // true
#"123456"  // true
#"098765"  // true
#"000000"  // true
#"090909"  // true

import re 

def pinChk(pinNum):
    strChk = re.sub('[0-9]','',pinNum)
    numChk = re.sub('[^0-9]','',pinNum)
    print(strChk)
    if len(strChk) == 0 and (len(numChk) == 4 or len(numChk) == 6):
        return True

    return False

if __name__ == "__main__":
    result = pinChk('023451')
    print(result)